addons/l10n_in_withholding/wizard/l10n_in_withhold_wizard.py

```diff
@@ -109,7 +109,7 @@ def _compute_company_id(self):
     def _compute_journal(self):
         for wizard in self:
             wizard.journal_id = wizard.company_id.l10n_in_withholding_journal_id or \
```

Comment:
Another option might be to keep the journal/account as optional on the company and check the parent companies here.  (but not sure which method is best yet)

Reply:
Okay 
Now it will check in the company and then check in the parent company.